src/Microsoft.ML.FastTree/TreeEnsemble/RegressionTree.cs

```diff
@@ -14,25 +14,25 @@
 using Microsoft.ML.Model;
 using Microsoft.ML.Model.Pfa;
 using Newtonsoft.Json.Linq;
-using Float = System.Single;
 
 namespace Microsoft.ML.Trainers.FastTree.Internal
 {
-    public class RegressionTree
+    /// Note that <see cref="RegressionTree"/> is shared between FastTree and LightGBM asemblies,
```

Comment:
>asemblies [](start = 87, length = 9)

typo

Reply:
Thanks! Let's do assemblies.

---
In reply to: [252031769](https://github.com/dotnet/machinelearning/pull/2243#discussion_r252031769) [](ancestors = 252031769)